crates/turbo-tasks-fetch/src/lib.rs

```diff
@@ -36,11 +36,32 @@ impl HttpResponseBody {
     }
 }
 
+#[turbo_tasks::value(shared)]
+#[derive(Debug)]
+pub enum ProxyOption {
+    None,
+    Http(String),
+    Https(String),
+}
```

Comment:
```suggestion
#[turbo_tasks::value(shared)]
#[derive(Debug)]
pub enum Proxy {
    Http(String),
    Https(String),
}
```

Reply:
Could probably also make this derive `TaskInput` and not have this be a turbo tasks value at all